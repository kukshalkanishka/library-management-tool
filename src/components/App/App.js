import React from "react";
import { BrowserRouter as Router, Route } from "react-router-dom";
import Options from "../Options/Options";
import Login from "../Login/Login";
import SignUp from "../SignUp/SignUp";
import Home from "../Home/Home";

const App = () => {
  return (
    <Router>
      <Route exact path="/" component={Options} />
      <Route path="/login" component={Login} />
      <Route path="/signup" component={SignUp} />
      <Route path="/home" component={Home} />
    </Router>
  );
};

export default App;
