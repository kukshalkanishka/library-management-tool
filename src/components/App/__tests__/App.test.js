import React from "react";
import { shallow, configure } from "enzyme";
import App from "../App";

import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("App", () => {
  it("should render properly", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
