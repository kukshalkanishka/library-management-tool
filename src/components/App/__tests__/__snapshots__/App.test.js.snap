// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render properly 1`] = `
"<BrowserRouter>
  <Route exact={true} path=\\"/\\" component={[Function: Options]} />
  <Route path=\\"/login\\" component={[Function: Login]} />
  <Route path=\\"/signup\\" component={[Function: SignUp]} />
  <Route path=\\"/home\\" component={[function]} />
</BrowserRouter>"
`;
